### DISCUSSION TITLE
Index Error in Landsatxplore

### DISCUSSION THREAD

**Discussion by kv-95** [yannforget/landsatxplore#79] [Q&A]
![image](https://user-images.githubusercontent.com/49032825/169015676-8a2d8c7c-5532-4c1c-bd19-de03b0536ca1.png)

I am getting the above-mentioned error while trying to log in using `EarthExplorer`.

**Comment by picknick1337** [comment]
Have you figured out a fix for this yet? Please do let me know if you have.

**Reply by kv-95** [reply]
No. I couldn't find a way around it. So instead, I used their own application to download the images. Apparently, USGS has a Java-based application to perform bulk download. Go through the following links. 

- https://dds.cr.usgs.gov/bulk
- https://eebulk.cr.usgs.gov/

**Reply by picknick1337** [reply]
I did find something just after I dropped that reply from [here](https://github.com/yannforget/landsatxplore/pull/75)
It works after you comment out all the occurences of _ncform_ within earthexplorer.py
Also this to avoid additional errors : https://github.com/yannforget/landsatxplore/pull/66/files

**Reply by kv-95** [reply]
That's great! Thanks for sharing.
